src/Storages/MergeTree/MergeTreeData.h

```diff
@@ -551,11 +550,19 @@ class MergeTreeData : public IStorage, public WithMutableContext
     MutableDataPartsVector tryLoadPartsToAttach(const ASTPtr & partition, bool attach_part,
                                                 ContextPtr context, PartsTemporaryRename & renamed_parts);
 
-
     /// If the table contains too many active parts, sleep for a while to give them time to merge.
     /// If until is non-null, wake up from the sleep earlier if the event happened.
+    /// The decision to delay or throw is made according to settings 'parts_to_delay_insert' and 'parts_to_throw_insert'.
```

Comment:
```suggestion
    /// The decision to delay or throw is made according to settings 'number_of_mutations_to_delay' and 'number_of_mutations_to_throw'.
```

typo

Reply:
But it's a comment for function `delayInsertOrThrowIfNeeded`.

Reply:
sorry my bad